ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -192,9 +192,9 @@ set_local_repo() {
     version="$(lsb_release -sr)"
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
-       [ -z "$(rpm_test_version)" ] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; then
+       [ -z "$CI_RPM_TEST_VERSION" ] &&
+       { [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] ||
+         [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; }; then
```

Comment:
This is re-introducing a bug that has been fixed on master.  This needs reverting.

Reply:
Must have been missed in the rebase.